packages/dds/merge-tree/src/perspective.ts

```diff
@@ -3,70 +3,87 @@
  * Licensed under the MIT License.
  */
 
-import { UnassignedSequenceNumber } from "./constants.js";
+import { LocalClientId, UnassignedSequenceNumber } from "./constants.js";
```

Comment:
it would be good to have unit test for this file, especially the parts you plan to change